pkg/agent/agent.go

```diff
@@ -213,12 +215,11 @@ func OnAgentResourceChange(agentResourceChangeHandler ConfigChangeHandler) {
 }
 
 func startAPIServiceCache() {
-	var checkStatus hc.CheckStatus
-
 	// register the update cache job
 	newDiscoveryCacheJob := newDiscoveryCache(false)
 	if !agent.cfg.IsUsingGRPC() {
-		checkStatus = agent.apicClient.Healthcheck
+		hc.RegisterHealthcheck(util.AmplifyCentral, "central", agent.apicClient.Healthcheck)
```

Comment:
Maybe this could/should be converted into a job as well?

Reply:
Maybe for another story